### PR TITLE
Op/leetcode day2

### PR DESCRIPTION
**Pull request by sainiop**
Enhancement required in both solutions

### CODE REVIEW

File: Leetcode/Python/3_reverseNumber.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""
+https://leetcode.com/problems/reverse-integer/
+
+Given a signed 32-bit integer x, return x with its digits reversed.
+If reversing x causes the value to go outside the signed 32-bit integer range [-231, 231 - 1], then return 0.
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: Nit:- 2**31

File: Leetcode/Python/3_reverseNumber.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+"""
+
+
+class Solution:
+    def reverse(self, x: int) -> int:
+        reversed = 0
+
+        if x < -2147483648 or x > 2147483647 or x == 0:
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: Magic numbers :- using hardcoded values within code is not preferred.
MIN_VALUE, MAX_VALUE are suggestions. would be good to define them at the top and use when needed

File: Leetcode/Python/3_reverseNumber.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            return int("0")
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: return 0?

File: Leetcode/Python/3_reverseNumber.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+        elif x > 0:
+            while x > 0:
+                reminder = x % 10
+                x = x // 10
+
+                reversed = reminder + reversed * 10
+            if reversed < -2147483648 or reversed > 2147483647:
+                return int("0")
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: how about changing with 
```
def is_valid(number):
    if number < MIN_VALUE or number > MAX＿VALUE or number: 
        return 0 
    return number   
```

File: Leetcode/Python/3_reverseNumber.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            return reversed
+
+        elif x < 0:
+            x = -x
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: abs(x)?

File: Leetcode/Python/3_reverseNumber.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+            while x > 0:
+                reminder = x % 10
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: nit: remainder

File: Leetcode/Python/3_reverseNumber.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+class Solution:
+    def reverse(self, x: int) -> int:
+        reversed = 0
+
+        if x < -2147483648 or x > 2147483647 or x == 0:
+            return int("0")
+
+        elif x > 0:
+            while x > 0:
```
Comment (aarifsk) [inline Leetcode/Python/3_reverseNumber.py]: how about 
```
binary_x = int(bin(x)[2:])
```

**Comment by aarifsk** [comment]
Okay to change file names to Num_Upper_Camel_Case.py?

File: Leetcode/Python/3_reverseNumber.py (hunk 3 above)
Comment (sainiop) [reply Leetcode/Python/3_reverseNumber.py]: return 0 giving None.

File: Leetcode/Python/3_reverseNumber.py (hunk 4 above)
Comment (sainiop) [reply Leetcode/Python/3_reverseNumber.py]: use of function would be better

**Pull request merged**